doc/asab/web/restapidocs.rst

```diff
@@ -0,0 +1,107 @@
+REST API Docs
+==============
+
+ASAB's API service generates a `Swagger documentation <https://swagger.io/specification>`_ which automatically shows all
+of your endpoints. However you will need to follow some steps to add summaries, tags and more.
+
+Before you start, make sure that you have a `REST API microservice <https://asab.readthedocs.io/en/latest/tutorial/web/chapter2.html>`_.
```

Comment:
This part of documentation assumes you can run ASAB application.

Reply:
Sorry, I don't get what you mean